project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/css/user_settings.css

```diff
@@ -0,0 +1,27 @@
+.user_settings_row {
```

Comment:
I would as this to the theme in order to:
- have access to variables and / or mixins
- use sass nesting

We also only have few separate css files within modules.